source/playground-databases.txt

```diff
@@ -0,0 +1,115 @@
+.. _vsce-playground-databases:
+
+==============================
+Create Databases & Collections
+==============================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-vsce-preview.rst
+
+You can create 
+:manual:`databases and collections </core/databases-and-collections/>` 
+using a MongoDB Playground. 
+
+Prerequisites
+-------------
+
+If you have not done so already, you must complete the following 
+prerequisites before you can create a database or collection with a 
+MongoDB Playground:
+
+- :ref:`Create a connection to a MongoDB deployment 
+  <vsce-connect-task>`.
+- :ref:`Activate the connection to the MongoDB deployment 
+  <vsce-activate-connection>`.
+
+Create a Database and Collection
+--------------------------------
+
+Once you connect to your deployment using MongoDB for VS code, use the left 
+navigation to: 
+
+1. Select an active connection and click the :icon-fa5:`plus` icon that 
+   appears.
+   
+#. A MongoDB playground automatically opens with a template form to create 
+   a database and a regular collection or a 
+   :ref:`time series collection <vsce-time-series>`. 
+
+#. Fill in ``const database`` and ``const collection`` with names of your 
+   choice for your database and collection.
+
+#. Uncomment the form of the collection you wish to create.
+
+#. Fill in the collection fields according to your collection's specifications.
+
+#. To run the playground, click the :guilabel:`Play Button` at the top right
+   of the VS code navigation bar.
+
+After running the playground, the left navigation will update with new database 
+listed under the active connection it was created. When the new database is 
+expanded the new collection can be found there.
```

Comment:
Let's make this last sentence active voice. Maybe something like:

"You can find the newly created collection by expanding the new database."

Reply:
Done.